Library/Homebrew/formula_path.sh

```diff
@@ -39,20 +53,35 @@ homebrew-prefix() {
   fi
 
   if [[ -z "${formula_exists}" &&
-        -z "${HOMEBREW_NO_INSTALL_FROM_API}" &&
-        -f "${HOMEBREW_CACHE}/api/formula.json" ]]
+        -z "${HOMEBREW_NO_INSTALL_FROM_API}" ]]
   then
-    formula_exists="$(
-      ruby -rjson <<RUBY 2>/dev/null
-puts 1 if JSON.parse(File.read("${HOMEBREW_CACHE}/api/formula.json")).any? do |f|
-  f["name"] == "${formula}"
-end
-RUBY
-    )"
+    if [[ -f "${HOMEBREW_CACHE}/api/parsed/formula_names.txt" ]] &&
+       grep -Fxq "${formula}" "${HOMEBREW_CACHE}/api/parsed/formula_names.txt"
+    then
+      formula_exists="1"
+    elif [[ -f "${HOMEBREW_CACHE}/api/parsed/formula_aliases.txt" ]]
+    then
+      while IFS="|" read -r alias_name real_name
```

Comment:
Could we use the default separator to avoid needing to set `IFS`?

Reply:
Default IFS includes newlines but I'll give it a try

Reply:
@Bo98 sucks. in that case: ignore it and leave it as-is!